.github/workflows/lint_javascript.yml

```diff
@@ -0,0 +1,20 @@
+name: Test javascript
+on:
```

Comment:
Hey @AlexLloyd0 could you please restrict these to run only against the main branch for all actions?

```yaml
on:
  push:
    branches:
    - main
    paths:
    - 'javascript/**'
  pull_request:
    branches:
    - main
    paths:
    - 'javascript/**'
```

Otherwise you end up double-building for the branch + the ff-merge to main, and we have limited numbers of runners for the google organization. /cc @taquitos

Reply:
Done - thanks for the feedback!